### DISCUSSION TITLE
Embedding a Julia variable value in Markdown ?

### DISCUSSION THREAD

**Discussion by gjunqueira-sys** [quarto-dev/quarto-cli#2211] [Q&A]
Hello all. Is there a way to embed the value of a julia variable in a markdown text?
I saw there seems to be a way to do that in python but i tried with julia and it doesnt seem to output the value.

I want to do something simple like this:

"This is a text markdown displaying a computational result $x"

where x is a julia variable.

**Comment by rgaiacs** [comment]
@gjunqueira-sys are you talking about the Observable variable?

**Comment by gjunqueira-sys** [comment]
Hello.. Yes I think I am.






________________________________
